src/fdc/MSXtar.hh

```diff
@@ -66,9 +66,12 @@ private:
 	void chroot(std::string_view newRootDir, bool createDir);
 
 private:
+	unsigned getStartCluster(const MSXDirEntry& entry) const;
+
 	SectorAccessibleDisk& disk;
 	MemBuffer<SectorBuffer> fatBuffer;
 
+	bool fat16;
```

Comment:
Detail: if there's no reason for a particular order for member variables (e.g. group related variables together) then prefer to order them from large to small.

In this case that means moving the `fat16` variable from line 74 to line 80. The reason for this is alignment and (internal) padding. The current patch will enlarge the size of the `MSXtar` object with 4 bytes (1 for the `bool`, 3 for padding to keep the next `unsigned` variable aligned). But when moving the variable, the size remains the same (the `bool` now consumes a byte that was already used for padding).